cirq/study/flatten_expressions.py

```diff
@@ -13,8 +13,7 @@
 # limitations under the License.
 """Resolves symbolic expressions to unique symbols."""
 
-from typing import (overload, Any, Callable, Dict, List, Optional,
-                    Tuple, Union)
+from typing import (overload, Any, Callable, Dict, List, Optional, Tuple, Union)
```

Comment:
Nit: Unnecessary parentheses.  Yapf can't remove them because it doesn't modify the token stream.